tests/e2e/project.spec.ts

```diff
@@ -64,6 +64,9 @@ test("navigate to project", async ({ page }) => {
 
   // Show rendered README.md contents.
   await expect(page.locator("text=Git test repository")).toBeVisible();
+
+  // Number of nodes tracking this project.
+  await expect(page.locator("text=3 nodes")).toBeVisible();
```

Comment:
added this, wdyt @sebastinez ? can't run it locally to be sure

Reply:
Yeah that works, to get the CI to pass you'll need to also update the `heartwood-version` file

```diff
diff --git a/tests/support/heartwood-version b/tests/support/heartwood-version
index 81e79f2..ce545a9 100644
--- a/tests/support/heartwood-version
+++ b/tests/support/heartwood-version
@@ -1 +1 @@
-97f122c4512fbc5ea0b04a11db01d0d2d0977527
+fd91cf4d28bd370a6c5beb9ca158e4d91432df25
```